src/app/models/sds-namespace.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * SDS Namespace representation, see
- * {@link https://ocs-docs.osisoft.com/Content_Portal/Documentation/Management/Account_Namespace_1.html|OCS Documentation},
+ * {@link https://ocs-docs.osisoft.com/Content_Portal/Documentation/Management/Account_Namespace_1.html|ADH Documentation},
```

Comment:
it looks like all of these links are not great to use anymore. If you go to any of them, it renders the OCS Content Portal for a few seconds, then it auto routes you to the OCS Documention main landing page. These are therefore effectively dead links.